### ISSUE TITLE
Perspective or symmetric brush.

### ISSUE THREAD

**Issue by prandygame** [opentoonz/opentoonz#4117]
Symmetrical brush: A symmetrical brush would be very useful when making structures, backgrounds with perspective, this is the idea, the brush paints in two or more places at the same time, each point can be closer to the original brush, you make a circle and draw several more circles at the selected distance, if you draw a circle the other circles could decrease in size and thickness creating a unique perspective brush never seen, I think, you can also choose if the symmetrical copies are drawn horizontally, diagonally or vertically, saving the time of creating several perspective windows, doors etc, well, the brush is symmetrical although it can be more than that, imagine that you have to create several windows, you draw one and others are drawn, but each one is smaller than the previous one, you modify the distance, the number of brushes, if you want it to be in depth or all of the same size, pressing shift could make straight lines, even each brush could paint with a different stroke, or color, would be a super brush, the perspective brush, do it before someone believes it, I think it doesn't exist yet haha.

**Comment by prandygame** [comment]
??

**Comment by RodneyBaker** [comment]
Are you looking for other ways to do this in Opentoonz?
If you are then perhaps we can detail those.

**Comment by prandygame** [comment]
> Are you looking for other ways to do this in Opentoonz? If you are then perhaps we can detail those.

Yes, the plan is that with that brush and other tools we can more easily create backgrounds with perspective, also put vanishing points with special rules, so that straight lines can be drawn to these points.

**Comment by RodneyBaker** [comment]
I mentioned this in your other feature request for Perapective Tools but...

The Tahoma2D fork has added (or is adding) a Symmetry Tool that works nicely.

Here's one of the tests I did with that tool:

![image](https://user-images.githubusercontent.com/2576432/221866602-51b57cf6-534f-4666-9951-f70c3348c145.png)

This one might be a little harder to tell where symmetry was used as it intentionally tries to hide that:

![scisymmetryC](https://user-images.githubusercontent.com/2576432/221867004-5e21b98d-cab7-4892-9b40-f045434aa0d8.gif)
